Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.11-slim
+
+LABEL org.opencontainers.image.source="https://github.com/neuralmagic/guidellm"
+LABEL org.opencontainers.image.description="GuideLLM Benchmark Container"
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Create non-root user
+RUN useradd -m -u 1000 guidellm
+
+# Set working directory
+WORKDIR /app
+
+# Install GuideLLM
+RUN pip install git+https://github.com/neuralmagic/guidellm.git
+
+# Copy and set up the benchmark script
+COPY run_benchmark.sh /app/
+RUN chmod +x /app/run_benchmark.sh
```

Comment:
I think this can be avoided if you do this on the code file itself, so you don't have to do it again inside the container.